pkg/trace/customtagsRC/custom_tags_rc_handler.go

```diff
@@ -0,0 +1,87 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package customtagsRC
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/remoteconfig/state"
+	"github.com/DataDog/datadog-agent/pkg/trace/config"
+	"github.com/DataDog/datadog-agent/pkg/trace/log"
+	"github.com/davecgh/go-spew/spew"
+)
+
+type CustomTagsConfig struct {
+	// array of fields - "span name: tag name"
+	CustomTagsArr *[]string `json:"custom_tags"`
```

Comment:
```suggestion
	CustomTagsArr []string `json:"custom_tags"`
```

Typically you don't see a pointer to a slice in Go. There's a difference between nil, and an empty slice, and you can use that to differentiate if a value isn't set.